pkg/logging/constants.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package logging supplies common constants to ensure consistent use of structured logs.
+package logging
+
+import (
+	"github.com/kcp-dev/logicalcluster/v2"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// ReconcilerKey is used to identify a reconciler.
+	ReconcilerKey = "reconciler"
+
+	// KeyKey is used to expose the workqueue key being processed.
+	KeyKey = "key"
```

Comment:
QueueKey

Reply:
QueueKeyKey ;-)

Reply:
maybe make a package loggingkeys. Reads more easily.

Reply:
On the other hand: do we really want these keys? Isn't `"name"` not good enough?